### PR TITLE
Adds error message whenever the default model is not found

### PR DESCRIPTION
**Pull request by kevinislas2**
I was following along the instructions in the printed book and couldn't get Chapter1 to run. After digging into the code (and eventually reading the repo's readme) I found about having to run the bootstrap command.

This pull request just adds a simple error message whenever that happens to hopefully help other readers if they get stuck on that part just like me.

### CODE REVIEW

File: source/raptor/foundation/gltf.hpp [hunk 1]
```diff
@@ -12,6 +12,9 @@ static const char* kDefault3DModel = "../deps/src/glTF-Sample-Models/2.0/Sponza/
         argv[1] = const_cast<char*>(kDefault3DModel);\
     }\
     else {\
+       printf("Unable to find default model. Please see "\
```
Comment (theWatchmen) [inline source/raptor/foundation/gltf.hpp:15]: Thanks for this! Could you rephrase it as
```
"Unable to find default model. Please check the README in the root folder and make sure you've run `python ./bootstrap.py` to download all the additional assets for this project.\n"
```

Comment (kevinislas2) [reply source/raptor/foundation/gltf.hpp]: Absolutely. I've pushed the changes with the updated error message. 

Thanks for the code review!
